<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>531</width>
     <height>411</height>
    </rect>
   </property>
   <property name="title">
    <string>Vertical Control</string>
   </property>
   <widget class="QSlider" name="vSlider_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(255, 0, 4);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vSlider_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(255, 0, 4);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vSlider_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(255, 0, 4);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vSlider_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(255, 0, 4);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="vSlider_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>51</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">gridline-color: rgb(255, 0, 4);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
